source/rules/literals.rs

```diff
@@ -239,7 +239,7 @@ fn string_single_quoted(input: &[u8]) -> Result<&[u8], Literal> {
         return Result::Error(Error::Code(ErrorKind::Custom(StringError::TooShort as u32)));
     }
 
-    if input[0] == 'b' as u8 {
+    if input[0] == 'b' as u8 || input[0] == 'B' as u8 {
```

Comment:
Only this condition is similar. The minimum required heuristic length is different.
This is likely to change when we will support string interpolation (see #3).

Do you see any easy factorisation?